### DISCUSSION TITLE
Nut Client wann wird runtergefahren

### DISCUSSION THREAD

**Discussion by warp735** [OpenCCU/OpenCCU#2376] [🇩🇪 Fragen&Antworten] [answered]
In der Doku heisst es: "erstes eine Alarmmeldung innerhalb des HomeMatic-Systems generieren und dann bei zu langem Ausfall auch RaspberryMatic automatisch geordnet herunterfahren."

Was ist denn ein langer Ausfall? Kann man das nicht irgendwo konfigurieren? Mein Nutmaster fährt runter, wenn die Batterie weniger wie 45min Laufzeit hat. Wird RM jetzt kurz vorher informiert oder läuft RM einfach weiter? Das wäre in sofern blöd, weil der Master dann ja nicht mehr erreichbar ist.

**Comment by jens-maus** [comment]
Probieren geht über studieren?!? ;)

**Comment by warp735** [comment]
toller Tipp... erstens will ich jetzt nicht zum Testen zig mal den Hauptserver runterfahren und zweitens fährt ioBroker auch meine Unifi Hardware runter, kurz bevor der Server abschaltet. Wie soll man da noch kontrollieren, ob RM heruntergefahren ist?

**Reply by jens-maus** [reply, marked as answer]
Im Grunde läuft das ganze mit dem NUT als Client oder Server nicht anders als bei jedem anderen Linux: du musst die jeweilige NUT Konfiguration auf deine speziellen Bedürfnisse anpassen.

D.h. Bei RaspberryMatic ist die reine Default Installation/Setup so, das da ausser einer WebUI Notification bzw Alarmmeldung in der WebUI keinerlei shutdown oder ähnliches passiert.

Du musst also erst selbst Hand anlegen und hier z.b. die `/etc/config/nut/nut_notify.sh` so anpassen das da dann in Abhängigkeit des jeweiligen Events ein shutdown passiert oder du machst das stattdessen via eines WebUI Programmes/Logik, da dort ja ohnehin die Alarmmeldungen dann auch auflaufen.

Anlehnen kannst du dich bei der NUT Konfigurationsänderung an die vielen NUT Dokumentationen die man im Internet findet. So z.b. die hier:

https://wiki.ubuntuusers.de/USV/NUT/

Und dann musst du eigentlich nur einen Blick in die existierenden Konfigdateien unter `/etc/config/nut` legen und z.b. die `nut_notify.sh` anhand deiner Bedürfnisse anpassen. Standardmäßig sieht die wie folgt aus:

https://github.com/jens-maus/RaspberryMatic/blob/master/buildroot-external/overlay/base-raspmatic/etc/config_templates/nut/nut_notify.sh

Und macht eben eine reine Alarmierung und mehr nicht.

Um was du jedoch IMHO nicht drumherum kommst ist, das ganze in der Tat am Ende ausgiebig zu testen wenn du sicherstellen willst das RaspberryMatic gar nicht oder gar zur falschen Zeut herunterfährt...

**Comment by Baxxy13** [comment]
Ziemlich umfangreiche Informationen sowie mögliche Konfigurationen bezüglich NUT und RaspberryMatic findet man z.B. [hier](https://github.com/wolwin/WW-mySHT/blob/master/SHT_RM_Nut/README.md)

**Comment by warp735** [comment]
Danke für die Antworten... dachte das wäre etwas "mehr" integriert und nicht einfach nur "mit drauf"

Habs wieder deaktiviert und schick den Shutdown von ioBroker aus. Merci
